### PR TITLE
Add docs for using custom fonts with Android paywalls

### PR DESCRIPTION
**Pull request by tonidero**
## Motivation / Description
This adds some extra documentation to the paywalls section explaining how to use custom fonts. 

We should hold this until a new release has been published with these new APIs

### CODE REVIEW

File: code_blocks/🛠 Tools/paywalls/paywalls_activity_single_google_font.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        )
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        paywallActivityLauncher = PaywallActivityLauncher(this, this)
+        ...
+    }
+
+    ...
```
Comment (NachoSoto) [inline code_blocks/🛠 Tools/paywalls/paywalls_activity_single_google_font.kt:16]: Hmm maybe remove this?

File: code_blocks/🛠 Tools/paywalls/paywalls_activity_single_resource_font.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        )
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        paywallActivityLauncher = PaywallActivityLauncher(this, this)
+        ...
+    }
+
+    ...
```
Comment (NachoSoto) [inline code_blocks/🛠 Tools/paywalls/paywalls_activity_single_resource_font.kt:15]: I think we don't do this in other snippets. If/when we start verifying this code it'll fail to compile.

File: docs_source/🛠 Tools/paywalls/displaying-paywalls.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ RevenueCat Paywalls will, by default, show paywalls fullscreen and there are mul
 
 - Depending on an entitlement with `PaywallDialog`
 - Custom logic with `PaywallDialog`
-- Manually with `PaywallView`, `PaywallDialog`, or `PaywallActivityLauncher`
+- Manually with `Paywall`, `PaywallDialog`, or `PaywallActivityLauncher`
```
Comment (NachoSoto) [inline docs_source/🛠 Tools/paywalls/displaying-paywalls.md:107]: Oh good catch!

**Review by NachoSoto — APPROVED** [review]
Awesome thanks!

File: code_blocks/🛠 Tools/paywalls/paywalls_activity_single_resource_font.kt (hunk 2 above)
Comment (tonidero) [reply code_blocks/🛠 Tools/paywalls/paywalls_activity_single_resource_font.kt:15]: Yeah, I can remove it 👍

**Pull request merged**